1_fetch.yml

```diff
@@ -0,0 +1,71 @@
+
+packages:
+  - aws.s3
+  - aws.signature
+  - sf
+  - maps
+  - lwgeom
+  - dplyr
+
+sources:
+  - 1_fetch/src/create_conus_sf.R
+  - 1_fetch/src/download_from_s3.R
+  - 1_fetch/src/fetch_date_task_plan.R
+
+targets:
+  all: 
+    depends: 
+      - 1_fetch/out/conus_sf.rds
+      - 1_fetch/out/hru.topojson
+      - 1_fetch/log/1_fetch_wbeep_storage_date_tasks.ind
+      
+  # Dates to visualize
+  # Dec 18 to Mar 10: 84 days (Jan 1-2 & Feb 6 threw an error for some reason so skipping)
```

Comment:
Maybe the tiles don't exist for these days?  Wasn't there an issue with some of the model driver data at the start of the new year?

Reply:
Yeah, that sounds right. Feb 6 was new to me, though!